system-addon/content-src/components/Sections/_Sections.scss

```diff
@@ -52,25 +62,9 @@
         text-align: center;
       }
     }
-  }
-}
-
-.wide-layout-enabled {
-  .sections-list {
-    .section-list {
-      @media (min-width: $break-point-widest) and (max-width: $break-point-widest + 2 * $card-width) {
-        :nth-child(3n) {
-          @include context-menu-open-left;
-        }
-      }
 
-      @media (min-width: $break-point-widest) {
-        grid-template-columns: repeat(auto-fit, $card-width-large);
-      }
+    @media (min-width: $break-point-widest) {
+      height: $card-height-large;
```

Comment:
Oh, was this previously always making things large if it was enabled?

Reply:
yup. oops. fixed now :)